src/System.Reflection.Emit.Lightweight/ref/System.Reflection.Emit.Lightweight.cs

```diff
@@ -8,6 +8,29 @@
 
 namespace System.Reflection.Emit
 {
+    public sealed partial class DynamicILInfo
+    {
+        internal DynamicILInfo(object scope, DynamicMethod method, byte[] methodSignature) { }
```

Comment:
Remove this. In our ref assembly code we only use `internal` (as opposed to possibly `protected internal`) in the single case where we want to prevent there being a default parameterless constructor. This constructor is not parameterless so it doesn't need to be here

Reply:
I added this because I wanted to prevent there being a default parameterless constructor.

Reply:
You should use internal constructor with no arguments. It is that pattern we use in reference assemblies everywhere in corefx.